manim/mobject/graphing/number_line.py

```diff
@@ -324,24 +324,40 @@ def get_tick_range(self) -> np.ndarray:
 
         return self.scaling.function(tick_range)
 
-    def number_to_point(self, number: float) -> np.ndarray:
+    def number_to_point(self, number: float | np.ndarray) -> np.ndarray:
```

Comment:
Not necessarily an `np.ndarray`, as shown by the example below this in the docstring, where a regular list is used.
```suggestion
    def number_to_point(self, number: float | Sequence[float]) -> np.ndarray:
```

Reply:
See comment above